hw3_task2.js

```diff
@@ -0,0 +1,23 @@
+import axios from 'axios';
+import fs from 'fs';
+
+async function processImage(imageUrl, filePath) {
+    try {
+        const response = await axios.get(imageUrl, {
+            responseType: 'arraybuffer'
+        });
+
+        fs.writeFileSync(filePath, response.data);
+
+        return Promise.resolve('Image successfully saved.');
+    } catch (error) {
+        return Promise.reject('Error saving image: ' + error.message);
+    }
+}
+
+const imageUrl = 'https://platinumlist.net/guide/wp-content/uploads/2023/03/IMG-worlds-of-adventure.webp';
+const filePath = 'images/image.jpg';
```

Comment:
when try to download image got error 
```
node hw3_task2.js 
Error saving image: ENOENT: no such file or directory, open 'images/image.jpg'
```
please add an ability to create folder during request or during server start function